### PR TITLE
Refactor/user

### PR DESCRIPTION
**Pull request by ramirobg94**
There are still some actions that we need to move. 
- initApp

The old app state was deleted with its reducer

useAuth, useUser and useProfilePicture were moved into user hooks

**we need to move this**
`_fetchUserAndSurvey`

### CODE REVIEW

File: webapp/store/user/actions.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import axios from 'axios'
+import { LoaderActions } from '@webapp/store/ui'
+
+export const APP_USER_LOGOUT = 'app/user/logout'
```
Comment (minotogna) [inline webapp/store/user/actions.js:4]: should this action be just: ```export const USER_LOGOUT = 'user/logout'```

File: webapp/app/actions.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
-export const appUserLogout = 'app/user/logout'
+
+const _fetchUserAndSurvey = async () => {
+  const {
+    data: { user, survey },
+  } = await axios.get('/auth/user')
+  return { user, survey }
+}
 
 // ====== INIT
```
Comment (ramirobg94) [inline webapp/app/actions.js:17]: this is going to be refactored to `initSystem` dispatching an action `SYSTEM_INIT = 'system/init'`

File: webapp/app/actions.js [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
 
 export const appPropsChange = 'app/props/change'
-export const appUserLogout = 'app/user/logout'
+
+const _fetchUserAndSurvey = async () => {
+  const {
+    data: { user, survey },
+  } = await axios.get('/auth/user')
+  return { user, survey }
+}
```
Comment (ramirobg94) [inline webapp/app/actions.js:14]: we need to find a place for this but maybe the `initSystem` should dispatch `initUser` which fetch survey and user too

Comment (minotogna) [reply webapp/app/actions.js:14]: ok in this case yes

**Pull request merged**